clients/java/qa/pom.xml

```diff
@@ -2,7 +2,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
   <modelVersion>4.0.0</modelVersion>
-  <name>Camunda External Task Client - QA</name>
+  <name>Camunda Platform - Java External Task Client - QA</name>
 
   <groupId>org.camunda.bpm.qa</groupId>
   <artifactId>qa</artifactId>
```

Comment:
Maybe we can rename this a bit closer to the external task client, so it does not create any clashes in the platform in the future? It seems to be fine for the time being, but `qa` is really very generic. 😄 
Note: If you change it, it must of course also be changed in the child modules.

Reply:
Isn't `Java External Task Client` already quite accurate? 🙂 Or do I not get your point? 🤔

Reply:
Oh sorry, I wasn't explicit enough I guess. It is the `artifactId` that I am worried about 😃

Reply:
Got it. You are right.